### DISCUSSION TITLE
item's ability

### DISCUSSION THREAD

**Discussion by johndoe71rus** [diasurgical/DevilutionX#1538] [Q&A] [answered]
Is there an increase in the effect if there are several items?
For example, is 2  items with speed / fast hit recovery better than one?
Or does one item improve the characteristics and the second no longer affects?

**Comment by AJenbo** [comment, marked as answer]
Depends on the type of ability, speed does not stack.

**Reply by johndoe71rus** [reply]
You could clarify.
Resistance stacks up.
Characteristics are added.
What about other effects?

**Reply by AJenbo** [reply]
Well since I don't know of the top of my head I would just have to look it up on google/wikis.

Jarulf's guide is really good for this sort of info: http://www.lurkerlounge.com/diablo/jarulf/jarulf162.pdf

**Reply by qndel** [reply]
We haven't changed the mechanics of the original game, why do you ask here instead of searching for diablo 1 guides?
